pkg/ingester/tailer.go

```diff
@@ -91,7 +92,10 @@ func (t *tailer) loop() {
 			tailResponse := logproto.TailResponse{Stream: stream, DroppedStreams: t.popDroppedStreams()}
 			err = t.conn.Send(&tailResponse)
 			if err != nil {
-				level.Error(util.Logger).Log("Error writing to tail client", fmt.Sprintf("%v", err))
+				// Don't log any error due to tail client closing the connection
+				if !util.IsConnCanceled(err) {
+					level.Error(cortex_util.Logger).Log("Error writing to tail client", fmt.Sprintf("%v", err))
```

Comment:
```suggestion
					level.Error(cortex_util.Logger).Log("msg","Error writing to tail client", "err", err)
```

Reply:
Can you take this opportunity to rework you all your log messages as per above ? This is how gokit is supposed to be used. `"k1","v1","k2","v2"`

Reply:
Sure, will do. Thanks for the review!